pynestml/codegeneration/resources_nest/point_neuron/setup/CMakeLists.txt.jinja2

```diff
@@ -175,37 +177,30 @@ execute_process(
     OUTPUT_STRIP_TRAILING_WHITESPACE
 )
 
-# Get the data install dir.
-execute_process(
-    COMMAND ${NEST_CONFIG} --datadir
-    RESULT_VARIABLE RES_VAR
-    OUTPUT_VARIABLE NEST_DATADIR
-    OUTPUT_STRIP_TRAILING_WHITESPACE
-)
-
-# Get the documentation install dir.
-execute_process(
-    COMMAND ${NEST_CONFIG} --docdir
-    RESULT_VARIABLE RES_VAR
-    OUTPUT_VARIABLE NEST_DOCDIR
-    OUTPUT_STRIP_TRAILING_WHITESPACE
-)
-
-# Get the library install dir.
-execute_process(
-    COMMAND ${NEST_CONFIG} --libdir
-    RESULT_VARIABLE RES_VAR
-    OUTPUT_VARIABLE NEST_LIBDIR
-    OUTPUT_STRIP_TRAILING_WHITESPACE
-)
-
 # on OS X
 set( CMAKE_MACOSX_RPATH ON )
 
-# Install all binaries to NEST's install directories.
-set( CMAKE_INSTALL_LIBDIR ${NEST_LIBDIR}/nest CACHE STRING "object code libraries (lib/nest or lib64/nest or lib/<multiarch-tuple>/nest on Debian)" FORCE )
-set( CMAKE_INSTALL_DOCDIR ${NEST_DOCDIR} CACHE STRING "documentation root (DATAROOTDIR/doc/nest)" FORCE )
-set( CMAKE_INSTALL_DATADIR ${NEST_DATADIR} CACHE STRING "read-only architecture-independent data (DATAROOTDIR/nest)" FORCE )
+
+if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
+  # Use the `NEST_PREFIX` as `CMAKE_INSTALL_PREFIX`.
+	set( CMAKE_INSTALL_PREFIX ${NEST_PREFIX} CACHE STRING "Install path prefix, prepended onto install directories." FORCE )
+  # Retrieve libs folder in nest
```

Comment:
This seems to repeat the code on lines 123-125 above. Is this by accident or by design?

Reply:
Ops, Yeah, it seems that I forgot to delete that block. Here I mean the block at 123-125.